extras/rya.console/src/main/java/mvm/rya/shell/command/mongo/administrative/MongoUninstall.java

```diff
@@ -48,9 +47,12 @@ public MongoUninstall(final MongoConnectionDetails connectionDetails, final Mong
     }
 
     @Override
-    public void uninstall(final String instanceName) throws InstanceDoesNotExistException, CommandException {
-        // TODO Auto-generated method stub
-
-        throw new UnsupportedOperationException("Not implemented yet.");
+    public void uninstall(final String ryaInstanceName) throws InstanceDoesNotExistException, CommandException {
+        try {
+            // TODO impl, test
+            getClient().dropDatabase(ryaInstanceName);
+        }catch (final MongoException e) {
```

Comment:
no, the only way to do that would be to get the list of database names and check for exists.  if not, throw expection.  I can do that if you want me to